observations/obs_converters/even_sphere/run_fixed_network_seq.csh

```diff
@@ -4,11 +4,11 @@
 # by UCAR, "as is", without charge, subject to all terms of use at
 # http://www.image.ucar.edu/DAReS/DART/DART_download
 
-# loop, calling ./create_fixed_network_seq to create separate
-# files for each time period.  edit the values below to change
-# the dates and intervals.
-
-# this one makes 2 files/day, 12 hours apart, single time per file.
+# Call ./create_fixed_network_seq to create a separate file for each time period.  
+# Edit the values below to change the dates and intervals.
+# It makes 2 files/day, 12 hours apart, single time per file.
```

Comment:
maybe mention 0Z and 12Z in this comment (if that's correct)

Reply:
I've changed the comments to reflect that files at 0 and 12 UTC are the default setup.